lang/en.json

```diff
@@ -144,8 +144,8 @@
         "or": "or",
         "compare": "Compare",
         "sign_up": "Sign Up",
-        "login": "Login",
-        "logout": "Logout",
+        "login": "Log in",
```

Comment:
I'm OK with "Sign in" and I much prefer "Register" over "Sign up". With Stencil, at least merchants and partners can now change these values if they have a stylistic preference, but we should provide best practice defaults.

I confess I'm a total noob at GitHub so I don't know how to change my own code. Is it easy to explain to me or should I drop by someone's desk tomorrow?